acedit/util.py

```diff
@@ -296,50 +296,29 @@ def run_solution(args):
             num_cases = len(os.listdir(testcases_path)) / 2
             results, expected_outputs, user_outputs = [], [], []
 
-            if extension == 'py':
-
-                for i in xrange(num_cases):
-                    status = os.system('cat ' + os.path.join(testcases_path, 'Input' + str(
-                        i)) + ' | timeout 3s python \'' + problem_path + '.py\' > temp_output' + str(i))
-                    if status == 124:
-                        # Time Limit Exceeded
-                        results += [Utilities.colors['BOLD'] +
-                                    Utilities.colors['YELLOW'] + 'TLE' + Utilities.colors['ENDC']]
-
-                    elif status == 0:
-                        # Ran successfully
-                        with open('temp_output' + str(i), 'r') as temp_handler, open(os.path.join(testcases_path, 'Output' + str(i)), 'r') as out_handler:
-                            expected_output = out_handler.read().strip().split('\n')
-                            user_output = temp_handler.read().strip().split('\n')
-
-                            expected_output = '\n'.join(
-                                [line.strip() for line in expected_output])
-                            user_output = '\n'.join(
-                                [line.strip() for line in user_output])
-
-                            expected_outputs += [expected_output]
-                            user_outputs += [user_output]
-
-                        if expected_output == user_output:
-                            # All Correct
-                            results += [Utilities.colors['BOLD'] + Utilities.colors[
-                                'GREEN'] + 'AC' + Utilities.colors['ENDC']]
-                        else:
-                            # Wrong Answer
-                            results += [Utilities.colors['BOLD'] +
-                                        Utilities.colors['RED'] + 'WA' + Utilities.colors['ENDC']]
-
-                    else:
-                        # Runtime Error
-                        results += [Utilities.colors['BOLD'] +
-                                    Utilities.colors['RED'] + 'RTE' + Utilities.colors['ENDC']]
-
-            elif extension in ['c', 'cpp', 'java']:
-
-                compiler = {'c': 'gcc', 'cpp': 'g++', 'java': 'javac -d .'}[extension]
-                execute_command = {'c': './a.out', 'cpp': './a.out', 'java': 'java ' + basename}[extension]
-                compile_status = os.system(
-                    compiler + ' \'' + problem_path + '.' + extension + '\'')
+            if extension in ['c', 'cpp', 'java', 'py', 'hs']:
```

Comment:
I think you missed `rb` in the list

Reply:
Yes, I haven't tested it, cauze it was too simple to add :)